src/views/AgmLocationChg.vue

```diff
@@ -425,12 +424,12 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
 
     // always include agm location change code
     // clear Priority flag and set the Waive Fees flag to true
```

Comment:
Should this comment be changed to reflect that the Waive Fees flag is no longer set to true?

Reply:
Yes! Fixed.